docs/content/troubleshooting/csi_ug_troubleshooting_logs.md

```diff
@@ -0,0 +1,23 @@
+# Log collection
+
+Use the CSI (Container Storage Interface) driver logs for problem identification.
+
+**Note:** These procedures are applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.
+
+To collect and display logs, related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands:
+
+## Log collection for CSI pods, daemonset, and StatefulSet
+
+`kubectl get all -n <namespace>  -l csi`
```

Comment:
this one isn't about logs
```suggestion
To collect and display status and logs, related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands:

## Status collection of CSI pods, daemonset, and statefulset

`kubectl get all -n <namespace> -l csi`
```

Reply:
@oriyarde @barakda  if this isn't log collection should I
1. move this status collection to the misc troubleshooting section?
or
2. rename the whole section to just "log and status" collection?

Reply:
I prefer 2.

also when we ask for "log collection", we mean status as well.
a common term for all of these could be "debug information"

Reply:
DONE

Reply:
it doesn't seem to be done here, but it was [indeed done in 1.7.0](https://github.com/IBM/ibm-block-csi-driver/pull/360/files/5bdd9dcccb1b39f1b6fc768275a981cc6a51eef6#diff-2e0d02fce1320e366e845f71164eb56f5ee63fe6074dc8aecb9c866fe5ed6886L9-R9). resolving